codejail_service/apps/api/v0/views.py

```diff
@@ -68,6 +68,42 @@
 # .. toggle_creation_date: 2025-01-30
 CODEJAIL_ENABLED = SettingToggle('CODEJAIL_ENABLED', default=False, module_name=__name__)
 
+# .. toggle_name: PYTHON_PATH_ONLY_ALLOW_KNOWN
```

Comment:
Do we need to make this configurable? If someone really needed/wanted to make this configurable, would it probably require an allow list? Same question for both toggles.

Reply:
I don't think so. If someone really wants this to be configurable, I think I'd want them to make a PR to add that functionality. Honestly, I'm not even sure this should be feature switched as it is.

Reply:
Ah, OK, understood now. Yes, happy to remove it.

Reply:
Toggles removed.